### PR TITLE
Ab#5536 plans page

### PR DESCRIPTION
**Pull request by additiverich**
- plans.html.jet (list of all plans) and plan information on associated curated page (curated.jet)
- [Zeplin plans](https://app.zeplin.io/project/607f961cf9f03790cc66e475/dashboard?sid=6184739dd625bf356fe5a0b5)
- needs s72-kibble 0.15.22 to get attributes like landscape images and expiry dates for plans
- some phrases will need translating
- extending page-content.jet and page-header.jet to take an optional `plan` parameter

### CODE REVIEW

**Comment by additiverich** [comment]
Going to tag in @stajs and @matthewthomsonnz - a lot of these changes were in the branch we just merged but here they are with some other folderol

File: package-lock.json [hunk 1]
```diff
@@ -1,8088 +1,8 @@
 {
   "name": "@shift72/core-template",
-  "version": "0.3.6",
-  "lockfileVersion": 2,
```
Comment (PeculiarGoat) [inline package-lock.json:4]: Think you need to upgrade your `npm`!

Comment (additiverich) [reply package-lock.json:4]: @PeculiarGoat What version are you using?

Comment (additiverich) [reply package-lock.json:4]: @PeculiarGoat Updated now

**Review by stajs — COMMENTED** [review]
I'm not understanding the comment that some of these changes have already been merged (? - if I even understand that correctly) and why GitHub is showing them as new changes, _however_ I'm willing to shrug it off and approve if someone (e.g. @matthewthomsonnz) can confirm that we've got these keys added to all the other language files too (shoddy Google Translate is acceptable).

**Review by stajs — APPROVED** [review]
Assuming that's all languages covered now.

**Pull request merged**